CODEOWNERS

```
@@ -5,5 +5,22 @@
 # Order is important: later rules override preceding rules
 
 # These owners will be the default owners for everything in the repo unless a later match takes precedence
-* @mesosphere-eng-noctilucent
-/stable/dispatch @devx
+* @mesosphere/sig-ksphere-cluster @mesosphere/sig-ksphere-multi-cluster
+/stable/argo-cd @mesosphere/devx
+/stable/awsebscsiprovisioner @mesosphere/sig-ksphere-cluster
+/stable/awsebsprovisioner @mesosphere/sig-ksphere-cluster
+/stable/azuredisk-csi-driver @mesosphere/sig-ksphere-cluster
+/stable/azurediskprovisioner @mesosphere/sig-ksphere-cluster
+/stable/defaultstorageclass @mesosphere/sig-ksphere-cluster
+/stable/dex @mesosphere/sig-ksphere-security
+/stable/dex-controller @mesosphere/sig-ksphere-security
+/stable/dispatch @mesosphere/devx
+/stable/gcpdisk-csi-driver @mesosphere/sig-ksphere-cluster
+/stable/gcpdiskprovisioner @mesosphere/sig-ksphere-cluster
+/stable/kommander @mesosphere/sig-ksphere-ui @mesosphere/sig-ksphere-multi-cluster
```

```suggestion
/stable/kommander @mesosphere/sig-ksphere-ui @mesosphere/sig-ksphere-multi-cluster @mesosphere/sig-ksphere-observability
```
They are at least working on it, shall we also make them owners?